packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.ssr.ts

```diff
@@ -14,7 +14,7 @@ import {queryReducer as query} from '../../features/query/query-slice.js';
 import {sortCriteriaReducer as sortCriteria} from '../../features/sort-criteria/sort-criteria-slice.js';
```

Comment:
Similar comment as above: this is the non-commerce param manager, we should also do the same thing with the commerce param manager